langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaStreamingChatModel.java

```diff
@@ -168,7 +170,7 @@ public OllamaStreamingChatModelBuilder stop(List<String> stop) {
             return this;
         }
 
-        public OllamaStreamingChatModelBuilder format(String format) {
+        public OllamaStreamingChatModelBuilder format(ResponseFormat format) {
```

Comment:
Same here

Reply:
Basically, the same answer as above, without the ChatRequest API part, as the streaming API does not have one for now.